quickbooks/2.0/modules/ROOT/pages/quickbooks-online-connector-reference.adoc

```diff
@@ -73,7 +73,21 @@ Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#
 | Stream Response a| Boolean |  Whether or not received responses should be streamed |  false |
 | Response Buffer Size a| Number |  The space in bytes for the buffer where the HTTP response will be stored. |  -1 |
 | Base Uri a| String |  Parameter base URI, each instance/tenant gets its own |  https://sandbox-quickbooks.api.intuit.com |
-| State a| String |  A String value that you send when the OAuth dance is initiated and that is returned back by the service provider with the token callback. |  security_token |
+| State a| String a|  A String value that you send when the OAuth dance is initiated and that is returned back by the service provider with the token callback.
+
+[NOTE]
+====
+The *State* parameter is typically standard for OAuth and is optional, however, QuickBooks requires the *State* parameter.
```

Comment:
Maybe make this less wordy:
Although the *State* parameter is optional, QuickBooks requires it.